Подключил:

constantineas opened the pull request:
Junit 4.6, Allure, commons-lang3

Добавил тестовые классы:
BaseTest (Класс - родитель, фактических тестов нет) BurgerConstructorTest
BurgerIngredientsPageTest
LoginPage
LogoutAccountTest
PersonalAccountTest
RegistrationPageTest

Сделал отчет Allure

Commit:
Подключил:
Junit 4.6, Allure, commons-lang3

Добавил тестовые классы:
BaseTest (Класс - родитель, фактических тестов нет)
BurgerConstructorTest
BurgerIngredientsPageTest
LoginPage
LogoutAccountTest
PersonalAccountTest
RegistrationPageTest

Сделал отчет Allure

nikolay-977 commented on src/main/java/pageObjects/AccountPage.java:
```diff
@@ -0,0 +1,140 @@
+package pageObjects;
```
Необходимо исправить наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы.

nikolay-977 commented on src/main/java/pageObjects/AccountPage.java:
```diff
@@ -0,0 +1,140 @@
+
+    // Локатор кнопки Войти в аккаунт
+    @FindBy(xpath = ".//button[text()='Войти в аккаунт']")
+    public WebElement buttonEnterAccount;
```
Необходимо исправить: Для веб-элементов используй модификатор доступа private.

Commit:
Для всех WebElement сделал модификатор доступа private

Изменил название пакета с pageObjects на pageobjects в соответствии с правилами нейминга

Применил форматирование CodeStyle Ctrl+Alt+L для всего кода, для лучше читаемости
